cli/tools/extractRegisterForReflection.java

```diff
@@ -0,0 +1,24 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS org.reflections:reflections:0.10.2
+//DEPS io.apicurio:apicurio-data-models:1.1.26
+
+import java.util.*;
+import org.reflections.*;
+
+import static java.lang.System.*;
+import static org.reflections.scanners.Scanners.*;
+
+public class extractRegisterForReflection {
```

Comment:
Why the lower case `e` in this class name?  That's pretty unusual in Java so I'm assuming it was intentional?

Reply:
Due to Java limitations, the name of the class should be strictly equal to the file name and I usually name scripts starting with a lowercase.

Given the fact that `jbang` is a pretty recent technology I doubt that there is any "strong" convention, in the front page they use lower-case scripts names though: https://www.jbang.dev/

Anyhow, not fussy about it, just suggest a preferred name and I'll fix it as you prefer 🙂